lib/demos/reference/motion_demo_container_transition.dart

```diff
@@ -0,0 +1,540 @@
+                        onTap: openContainer,
+                        title: Text(
+                          GalleryLocalizations.of(context)
+                                  .demoMotionListTileTitle +
+                              ' ${index + 1}',
+                        ),
+                        subtitle: Text(
+                          GalleryLocalizations.of(context)
+                              .demoMotionPlaceholderSubtitle,
+                        ),
+                      );
+                    },
+                  );
+                }),
+              ],
+            ),
+            floatingActionButton: OpenContainer(
+              transitionType: _transitionType,
+              openBuilder: (context, openContainer) {
+                return const _DetailsPage();
+              },
+              closedElevation: 6.0,
+              closedShape: const RoundedRectangleBorder(
+                borderRadius: BorderRadius.all(
+                  Radius.circular(_fabDimension / 2),
+                ),
+              ),
+              closedColor: Theme.of(context).colorScheme.secondary,
+              closedBuilder: (context, openContainer) {
+                return SizedBox(
+                  height: _fabDimension,
+                  width: _fabDimension,
+                  child: Center(
+                    child: Icon(
+                      Icons.add,
+                      color: Theme.of(context).colorScheme.onSecondary,
+                    ),
+                  ),
+                );
+              },
+            ),
+          ),
+        );
+      },
+    );
+  }
+}
+
+class _OpenContainerWrapper extends StatelessWidget {
+  const _OpenContainerWrapper({
+    this.closedBuilder,
+    this.transitionType,
+  });
+
+  final CloseContainerBuilder closedBuilder;
+  final ContainerTransitionType transitionType;
+
+  @override
+  Widget build(BuildContext context) {
+    return OpenContainer<bool>(
+      transitionType: transitionType,
+      openBuilder: (context, openContainer) {
+        return const _DetailsPage();
+      },
+      tappable: false,
+      closedBuilder: closedBuilder,
+    );
+  }
+}
+
+class _ExampleCard extends StatelessWidget {
+  const _ExampleCard({this.openContainer});
+
+  final VoidCallback openContainer;
+
+  @override
+  Widget build(BuildContext context) {
+    return _InkWellOverlay(
+      openContainer: openContainer,
+      height: 300,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: <Widget>[
+          Expanded(
+            child: Container(
+              color: Colors.black38,
+              child: Center(
+                child: Image.asset(
+                  'assets/demos/motion/placeholder_image.png',
+                  width: 100,
+                ),
+              ),
+            ),
+          ),
+          ListTile(
+            title: Text(
+                GalleryLocalizations.of(context).demoMotionPlaceholderTitle),
```

Comment:
you can do `final localizations = GalleryLocalizations.of(context)` at the top of the build, and reuse here to make this smaller

Reply:
Thanks! did it.